packages/generate/src/gel-prisma.ts

```diff
@@ -272,7 +272,7 @@ function processLinks(types: JSONType[], modules: string[]): ProcessedSpec {
 
           link_objects.push(lobj);
           link["has_link_object"] = true;
-          objtype["backlinks"][-1]["has_link_object"] = true;
+          objtype["backlinks"].at(-1)!["has_link_object"] = true;
```

Comment:
@vpetrovykh This was the one major change I made across this whole refactor: was this the intention of this code path? I think you _can_ use a negative index on an array:

```
> const a = [];
undefined
> a[-1] = "beep";
'beep'
> a
[ '-1': 'beep' ]
```

But I doubt that's what you were _trying_ to do here?

Reply:
Yeah, your fix is correct, I meant the last element of the array, not the actual index '-1'.